src/components/NewCommentForm.tsx

```diff
@@ -1,37 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Post } from '../types/Post';
+import { addComment } from '../api/services';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  selectedPost: Post | null;
+  addNewComment(commentFromServer: Comment): void;
+  onSetError(): void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  selectedPost,
+  addNewComment,
+  onSetError,
+}) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [comment, setComment] = useState('');
+
+  // const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const [nameError, setNameError] = useState(false);
+  const [emailError, serEmailError] = useState(false);
+  const [commentError, setCommentError] = useState(false);
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
thanks